content/docs/command-reference/import.md

```diff
@@ -177,12 +177,20 @@ When using this option, the import `.dvc` file will also have a `rev` subfield
 under `repo`:
 
 ```yaml
+md5: 7ff366c716a376ec009054f1c141dc17
+frozen: true
 deps:
-  - path: data/data.xml
-    repo:
-      url: git@github.com:iterative/dataset-registry.git
-      rev: cats-dogs-v1
-      rev_lock: 0547f5883fb18e523e35578e2f0d19648c8f2d5c
+- path: use-cases/cats-dogs
+  repo:
+    url: git@github.com:iterative/dataset-registry.git
+    rev: cats-dogs-v1
+    ^^^^^^^^^^^^^^^^^
```

Comment:
it's not a valid YAML syntax anymore, it might confuse people I think, Emre ... if we want to highlight the diff - we can do diff as we do in some other places, wdyt?

Reply:
It might be good, but using diff also doesn't lead to proper YAML. 

It looks it's already merged, but I can replace this with a `git` +/- example. DYT it will look better?

Reply:
may be :) I'm missing some context I think ... what is the reason to include it in the first place, btw? Looks like this current fix is enough, no?

Reply:
The surrounding text mentions how `--rev` changes the `.dvc` file contents, but the content itself is long to see. I've added `^^^` under `rev:` because of this. I don't think the user is expecting a valid YAML at this point, it's obvious that we're making them to notify the difference.

Reply:
I see what you were going for @iesahin but currently the YAML block is shown before using the `update` cmd so it wouldn't make sense to have a git-style diff here unless you rearrange that somehow. Feel free to open a PR suggestion. Thanks